inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/InAppMessageBaseView.kt

```diff
@@ -100,10 +102,15 @@ internal open class InAppMessageBaseView(context: Context, attrs: AttributeSet?)
     /**
      * This method binds data to buttons.
      */
+    @SuppressWarnings("LongMethod")
     private fun bindButtons() {
         // Set onClick listener to close button.
         val closeButton = findViewById<ImageButton>(R.id.message_close_button)
-        closeButton?.setOnClickListener(this.listener)
+        if (isDismissable) {
+            closeButton?.setOnClickListener(this.listener)
```

Comment:
can use `?.let` to remove multiple null checking

Reply:
done